applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -59,6 +106,15 @@ int KratosGeoSettlement::RunStage(const std::filesystem::path&            rWorki
         KRATOS_INFO("KratosGeoSettlement") << "Read the materials from " << material_file_path << std::endl;
     }
 
+    std::vector<std::shared_ptr<Process>> processes = GetProcesses(project_parameters);
+    std::vector<std::weak_ptr<Process>> process_observables;
+    process_observables.insert(process_observables.end(), processes.begin(), processes.end());
```

Comment:
I think you can do this also in one go, like this:
```suggestion
    std::vector<std::weak_ptr<Process>> process_observables{processes.begin(), processes.end()};
```

Reply:
Even shorter, nice!